packages/monaco-editor-wrapper/src/editorAppBase.ts

```diff
@@ -77,14 +88,14 @@ export abstract class EditorAppBase {
         return this.diffEditor;
     }
 
-    protected async createEditor(container: HTMLElement, editorOptions?: editor.IStandaloneEditorConstructionOptions): Promise<void> {
-        this.editor = createConfiguredEditor(container, editorOptions);
-        await this.updateEditorModel();
-    }
-
-    protected async createDiffEditor(container: HTMLElement, diffEditorOptions?: editor.IStandaloneDiffEditorConstructionOptions): Promise<void> {
-        this.diffEditor = createConfiguredDiffEditor(container, diffEditorOptions);
-        await this.updateDiffEditorModel();
+    async createEditors(container: HTMLElement): Promise<void> {
```

Comment:
No longer protected?

Reply:
`createEditors` was called by the extending classes which had another method. Now there is this unified one which is called by the wrapper. That change is intended.